script/dashboard-dev

```diff
@@ -55,27 +60,20 @@ run() {
 package main
 
 import (
-    "bytes"
-    "io"
-    "os"
+	"io/ioutil"
+	"os"
 )
 
 func Asset(path string) ([]byte, error) {
-    file, err := os.Open(path)
-    if err != nil {
-        return nil, err
-    }
-    var buf bytes.Buffer
-    _, err = io.Copy(&buf, file)
-    return buf.Bytes(), err
+	return ioutil.ReadFile(path)
 }
 
 func AssetInfo(path string) (os.FileInfo, error) {
-    return os.Stat(path)
+	return os.Stat(path)
 }
 BINDATA
```

Comment:
We possibly don't need this. I'll double check and see if any of the compile steps overwrite the checked-in version of `bindata.go` (this was previously the case).

Reply:
Confirmed. This isn't needed.